test/distributed/test_data_parallel.py

```diff
@@ -95,6 +96,26 @@ def test_parallel_apply(self):
             for out, expected in zip(outputs, expected_outputs):
                 self.assertEqual(out, expected)
 
+    @unittest.skipIf(not TEST_MULTIGPU, "multi-GPU not supported")
+    def test_parallel_apply_autocast(self):
+        l1 = nn.Linear(10, 5).to("cuda:0", torch.float)
+        l2 = nn.Linear(10, 5).to("cuda:1", torch.float)
+        i1 = torch.randn(2, 10, device="cuda:0", dtype=torch.float)
+        i2 = torch.randn(2, 10, device="cuda:1", dtype=torch.float)
+        with autocast():
+            expected1 = l1(i1)
+            expected2 = l2(i2)
+        modules = (l1, l2)
+        expected_outputs = (expected1, expected2)
+
+        # each input can be either a collection of positional arguments
+        #                       or an object representing the single argument
```

Comment:
any reason for the long indent before "or"?

Reply:
thanks for pointing this out. It also looks strange to me, but I do not know the reason, I am following [the example](https://github.com/pytorch/pytorch/blob/ccc831ae358278a0699df9aea236234b42bf2180/test/distributed/test_data_parallel.py#L91-L92). If it is not right, please help fix.

Reply:
that might be some debug vestige. No worries, I will land this as is, and submit a PR to fix both.